src/components/DetailSummary.js

```diff
@@ -56,6 +56,18 @@ class DetailSummary extends Component {
         getItem(baseName, routeId)
     }
 
+    getLabelForEntity(baseName) {
+        switch (baseName) {
+            case USER_GROUP:
+                return i18n.t('Edit user group')
+            case USER_ROLE:
+                return i18n.t('Edit user role')
+            case USER:
+            default:
```

Comment:
Having both default and USER for the same label seems confusing to me. I'd either omit the default case, or set the default to an appropriate fallback (empty string, null, etc.).